InnerEye/ML/SSL/ssl_augmentation_config.py

```diff
@@ -0,0 +1,85 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+from yacs.config import CfgNode
+
+config = CfgNode()
+
+config.preprocess = CfgNode()
+config.preprocess.center_crop_size = 224
+config.preprocess.resize = 256
```

Comment:
one line comment to add:
"images are first resized to 256x256"
"Then at the end center cropped to 224x224"